packages/rocketchat-lib/lib/callbacks.js

```diff
@@ -76,6 +76,12 @@ RocketChat.callbacks.remove = function(hookName, id) {
 RocketChat.callbacks.run = function(hook, item, constant) {
 	const callbacks = RocketChat.callbacks[hook];
 	if (callbacks && callbacks.length) {
+
+		let rocketchatHooksEnd;
+		if (Meteor.isServer) {
+			rocketchatHooksEnd = RocketChat.metrics.rocketchatHooks.startTimer({hook, callbacks_length: callbacks.length});
```

Comment:
I would like to have this as `RocketChat.statsTracker` : 
- `hooks` are each registered `callback`, i.e.: `sendNotificationsOnMessage`
- `callbacks` are named callbacks, i.e.: `beforeSaveMessage`